### ISSUE TITLE
processId can be larger than SMALLINT(5)

### ISSUE THREAD

**Issue by intagaming** [LegendOfMCPE/Hormones#36]
Happened when run the server (the actual server OS) for the long time, seen as below.
Quick workaround: Navicat -> Design Table -> int(6) will do the trick BUT according to this thread 
[https://stackoverflow.com/questions/17868218/what-is-the-maximum-process-id-on-windows](url)
this can be up to 4 billions, equals to UNSIGNED int(11). So just remind a little.

![image](https://user-images.githubusercontent.com/16471365/32783751-6cb531fc-c97f-11e7-8e1b-dadb7fff516d.png)

**Comment by SOF3** [comment]
Right, I didn't know about that. I also found that process ID on Linux can be the same modified recently.
